lms/static/js/views/fields.js

```diff
@@ -325,6 +325,7 @@
                     mode: this.mode,
                     title: this.options.title,
                     screenReaderTitle: this.options.screenReaderTitle || this.options.title,
+                    titleNotVisible: this.options.titleNotVisible || false,
```

Comment:
Can we use an attribute `titleVisible` which is true by default instead of `titleNotVisible`? Its hard to read negative bools.